packages/micromark-util-events-to-acorn/dev/index.js

```diff
@@ -186,6 +186,15 @@ export function eventsToAcorn(events, options) {
     for (const token of tokens) {
       fixPosition(token)
 
+      // Prefix and suffix should be ignored
+      if (
+        token.start === token.end &&
+        ((prefix && prefix.includes(token.type.label)) ||
+          (suffix && suffix.includes(token.type.label)))
```

Comment:
I don’t understand these two lines? My guess is that they’re not needed, that L191 is enough: micromark itself never creates empty tokens (as in: where `start === end`). So if they’re empty, they must be made empty by the position fixing?

Reply:
`eof` from acorn is empty.

Reply:
Ahh these are acorn tokens...
Could it be that that's the only one?

Reply:
I'm not sure, so I use the current implementation, it should cover common cases.

Reply:
Could you try? Perhaps also try the change in your node modules for eslint-mdx.
My guess is that it's *very likely* only EOF.
And otherwise we'll know soon enough.
And this current code looks a bit magical to me 😬